pkg/blockdevice/lvm/logical_volumes.go

```diff
@@ -75,12 +75,14 @@ func (lv *LogicalVolume) parseArgs(args string) (string, error) {
 
 	// Initialize the VarHandler substitution variables
 	varHandler := var_handler.NewVarHandler(map[string]string{
-		"$DEVICE_NUM":  fmt.Sprintf("%d", len(deviceNames)),
-		"$DEVICE_LIST": strings.Join(deviceNames, " "),
-		"$VG_NAME":     lv.VolumeGroup.Name,
-		"$LV_NAME":     lv.Name,
-		"$LV_SIZE":     fmt.Sprintf("%vK", (lv.Size / 1000)),
-		"$PERCENT_VG":  fmt.Sprintf("%v", lv.PercentVG) + "%VG",
+		"$DEVICE_NUM":   fmt.Sprintf("%d", len(deviceNames)),
+		"$DEVICE_NUM-1": fmt.Sprintf("%d", len(deviceNames)-1),
```

Comment:
This looks strange. A comment about what DEVICE_NUM-[1,2] are for would be helpful